google/iam.go

```diff
@@ -0,0 +1,96 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/cloudresourcemanager/v1"
+	"log"
+	"time"
+)
+
+type ResourceIamUpdater interface {
+	GetResourceIamPolicy() (*cloudresourcemanager.Policy, error)
+	SetProjectIamPolicy(policy *cloudresourcemanager.Policy) error
```

Comment:
Since this is the resource-agnostic file, should this be called SetResourceIamPolicy? (or SetIamPolicy or the like?)

Reply:
Ooops. I let this one slipped. Good catch